### PR TITLE
Moved files

### PR DESCRIPTION
**Pull request by ivan-guo**

### CODE REVIEW

**Comment by ivan-guo** [comment]
@straw-hat-gupta I moved a bunch of the files so we know which ones are the new ones - I'm still combining the significant strike files. Can you double check that the correct files are in the correct folders? (features is the new folder, old_feature_files has the uncombined features)

**Comment by straw-hat-gupta** [comment]
> Owner

will do

File: features/taped_stats.py
Comment (straw-hat-gupta) [inline features/taped_stats.py:1]: rename with _features at end for consistancy with all other files

**Review by straw-hat-gupta — APPROVED** [review]
lgtm, we are just missing the division features. the division feats are mentioned on the google doc

File: features/taped_stats.py
Comment (ivan-guo) [reply features/taped_stats.py:1]: Where are the division features?

Comment (ivan-guo) [reply features/taped_stats.py:1]: Oh nvm they are just renamed original columns, we don't actually need a file for them

**Pull request merged**